addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.js

```diff
@@ -141,6 +141,10 @@ export class DiscussSidebarCategories extends Component {
     }
 
     async toggleCategory(category) {
+        if (!category.serverStateKey) {
+            category.openLocally = !category.openLocally;
+            return;
+        }
         this.store.settings[category.serverStateKey] =
```

Comment:
Could define a `set open()`, so that `openLocally`/`serverStateKey` is all internal of this model.
This would allow to "clean" code just for some internal code of a model in the future :)